ci-scripts/docker/Dockerfile.mme.ci.rhel8

```diff
@@ -89,8 +89,8 @@ RUN ldconfig -v
 
 # Copy pre-built binaries for MME and SCTPD
 WORKDIR /magma-mme/bin
-COPY --from=magma-mme-builder $C_BUILD/oai/oai_mme/mme oai_mme
-COPY --from=magma-mme-builder $C_BUILD/sctpd/sctpd .
+COPY --from=magma-mme-builder $C_BUILD/core/oai/oai_mme/mme oai_mme
+COPY --from=magma-mme-builder $C_BUILD/sctpd/src/sctpd .
```

Comment:
@rdefosse is this the line that looks for sctpd? Is there a way to build this locally so I can test locall?